Assets/MRTK/SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -273,7 +273,15 @@ public string PointerName
         }
 
         /// <inheritdoc />
-        public IMixedRealityInputSource InputSourceParent { get; protected set; }
+        public IMixedRealityInputSource InputSourceParent
+        {
+            get { return base.Controller?.InputSource; }
+
+#if UNITY_2020_3_OR_NEWER
+            [Obsolete("Setting the Input Source Parent directly is no longer supported")]
+#endif
+            protected set { Debug.LogWarning("Setting the Input Source Parent directly is no longer supported"); }
```

Comment:
nit: could this string be refactored out as const, since it seems to be used multiple times?

Reply:
I think it's fine to leave it like this, it's similar to how we have some redundancy with Tooltip() and the function summary

Reply:
That's fair! I was thinking we at least had an opportunity to use the pattern here.
Whereas the summary/tooltip redundancy is unfortunately required, since I'm not aware of a way to expand a variable into a summary block (but that would be useful!)